superset-frontend/spec/javascripts/datasource/DatasourceEditor_spec.jsx

```diff
@@ -142,4 +147,65 @@ describe('DatasourceEditor', () => {
       wrapper.find(Field).find({ fieldKey: 'fetch_values_predicate' }).exists(),
     ).toBe(true);
   });
+
+  describe('enable edit Source tab', () => {
+    beforeAll(() => {
+      isFeatureEnabledMock = jest
+        .spyOn(featureFlags, 'isFeatureEnabled')
+        .mockImplementation(
+          feature => feature === 'ENABLE_DATASET_SOURCE_EDIT',
```

Comment:
I think this should be updated to `DISABLE_DATASET_SOURCE_EDIT` as well.

Why did the test pass even when the feature flag is not correct?

Reply:
fixed. The logic here is **_if feature is not enabled_**, though used the wrong feature name, this feature "disable edit", is not enabled.